src/client/Ruby/ms-rest/lib/ms_rest/http_operation_request.rb

```diff
@@ -121,7 +117,20 @@ def to_json(*a)
         log: log  
       }.to_json(*a)
     end
-    
+
+    def self.create_faraday_connection(base_uri, options = {})
```

Comment:
Any thoughts for why `create_faraday_connection` being a class method instead of instance method?